subprojects/core/src/main/java/org/gradle/execution/plan/DefaultExecutionPlan.java

```diff
@@ -610,6 +614,7 @@ public Node selectNext(WorkerLeaseRegistry.WorkerLease workerLease, ResourceLock
         for (Iterator<Node> iterator = dependenciesWhichRequireMonitoring.iterator(); iterator.hasNext();) {
             Node node = iterator.next();
             if (node.isComplete()) {
+                LOGGER.debug("Monitored node {} completed. Updating dependencies.", node);
```

Comment:
Nit: using whole sentences in debug logs seems weird to me. I don't think we do it anywhere else.

Reply:
What would you expect as a log message here?

Reply:
I changed it to not be two whole sentences.